plugin/advancedsessions/src/AdvancedSessionsPlugin.class.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+/**
+ * The Advanced Session allow add sessions' extra fields 
+ * @author Angel Fernando Quiroz Campos <angel.quiroz@beeznest.com>
+ * @package chamilo.plugin.advancedSessions
+ */
+class AdvancedSessionsPlugin extends Plugin
+{
+
+    const FIELD_NAME = 'as_description';
```

Comment:
"as" should be more expressive, like "advsess" or something like that, because as is an English word